hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancerTask.java

```diff
@@ -340,12 +344,13 @@ public List<ContainerBalancerTaskIterationStatusInfo> getCurrentIterationsStatis
         .mapToInt(ContainerBalancerTaskIterationStatusInfo::getIterationNumber)
         .max()
         .orElse(0);
```

Comment:
Wouldn't the last index in the list be the latest iteration? Why do we need to scan through it?